### ISSUE TITLE
CHECKOUT

### ISSUE THREAD

**Issue by florencebybike** [woocommerce/woocommerce-paypal-payments#1864]
### Describe the Bug

We configured our shop online with woocommerce and packlink pro on wordpress.
at the checkout, the page where you insert your phone number, e-mail and shipping address is missing.
instead there is written "You have to enter your address first in order to search for Drop-Off location. "
### To Reproduce

1. Go to '…' https://www.florencebybike.it/il-negozio/negozio/
2. Click on '…' something to put on the shopping bag
3. Scroll down to …' close the order. choose the shipping company.

### Screenshots

![you have to enter](https://github.com/woocommerce/woocommerce-paypal-payments/assets/151947830/2edac327-9bc1-45d4-9596-f9df892e20cb)

### Environment

 - WordPress Version
 - WooCommerce Version
 - Plugin Version
 - Browser [e.g. Chrome, Safari] and Version
 - Any other plugins installed

### Additional Details

<details><summary>System status</summary>

  ```
  <!--If applicable, paste the system status here. Please ensure you redact or remove any identifying information. -->
  ```

</details>

**Comment by InpsydeNiklas** [comment]
Hi @florencebybike this message on your checkout page should be unrelated to the WooCommerce PayPal Payments plugin. Or does this message disappear when you disable PayPal Payments? It seems more likely to me that this message comes from the Packlink Pro plugin. Have you tried reaching out to the [Packlink support](https://woo.com/vendor/packlink-shipping/)?

**Comment by florencebybike** [comment]
hello,
thank you!
if i disable paypal payments, nothing changes.
we talked to packlink pro support via phone and they said that this had 
to do with word press / woo commerce.
what else do you need to know yo help us further?
thank you very much.

Il 24.11.2023 18:30 Niklas Gutberlet ha scritto:

**Comment by InpsydeNiklas** [comment]
Thanks for checking! This GitHub repository is only about the PayPal integration for WooCommerce, and that's all we can help with.
According to the WooCommerce website, the Packlink Pro plugin for WooCommerce is supported directly by Packlink support team: https://woo.com/products/packlink-pro/
You can contact the WooCommerce support or the Packlink support best from the WooCommerce homepage here (needs a Woo account): https://woo.com/my-account/create-a-ticket/

But after a short research, I found a discussion in the wordpress.org plugin support forums regarding the message you mentioned: https://wordpress.org/support/topic/please-select-drop-off-location/
This may help you find a solution!
I hope this information guides you to a solution, and I will close this issue as there is little more we can help with in this regard.

**Comment by InpsydeNiklas** [comment]
Hi @florencebybike, one additional detail that may be relevant for you: Updating the plugin to the latest version should resolve the problem on your site. Right now, your site appears to be using Packlink 3.3.1 and this issue was fixed in version 3.3.2: https://wordpress.org/plugins/packlink-pro-shipping/#developers

**Comment by wheeleran** [comment]
no it wasn't fixed
